website/docs/r/pinpoint_app.markdown

```diff
@@ -0,0 +1,72 @@
+---
+layout: "aws"
+page_title: "AWS: aws_pinpoint_app"
+sidebar_current: "docs-aws-resource-pinpoint-app"
+description: |-
+  Provides a Pinpoint App resource.
+---
+
+# aws_pinpoint_app
+
+Provides a Pinpoint App resource.
+
+## Example Usage
+
+```hcl
+resource "aws_pinpoint_app" "example" {
+  name         = "test-app"
+  
+  limits {
+    maximum_duration    = 600
+  }
+
+  quiet_time {
+    start = "00:00"
+    end   = "06:00"
+  }
+}
+```
+
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Optional) The application name. By default generated by Terraform
+* `name_prefix` - (Optional) The name of the Pinpoint application. Conflicts with `name`
+* `campaign_hook` - (Optional) The default campaign limits for the app. These limits apply to each campaign for the app, unless the campaign overrides the default with limits of its own
+* `limits` - (Optional) The default campaign limits for the app. These limits apply to each campaign for the app, unless the campaign overrides the default with limits of its own
+* `quiet_time` - (Optional) The default quiet time for the app. Each campaign for this app sends no messages during this time unless the campaign overrides the default with a quiet time of its own
+
+`campaign_hook` supports the following:
+
+* `lambda_function_name` - (Optional) Lambda function name or ARN to be called for delivery. Conflicts with `web_url`
+* `mode` - (Required if `lambda_function_name` or `web_url` are provided) What mode Lambda should be invoked in. Valid values for this parameter are `DELIVERY`, `FILTER`.  
+* `web_url` - (Optional) Web URL to call for hook. If the URL has authentication specified it will be added as authentication to the request. Conflicts with `lambda_function_name`
+
+`limits` supports the following:
+
+* `daily` - (Optional) The maximum number of messages that the campaign can send daily. 
+* `maximum_duration` - (Optional) The length of time (in seconds) that the campaign can run before it ends and message deliveries stop. This duration begins at the scheduled start time for the campaign. The minimum value is 60.
+* `messages_per_second` - (Optional) The number of messages that the campaign can send per second. The minimum value is 50, and the maximum is 20000.
+* `total` - (Optional) The maximum total number of messages that the campaign can send.
+
+`quiet_time` supports the following:
+
+* `end` - (Optional) The default end time for quiet time in ISO 8601 format. Required if `start` is set
+* `start` - (Optional) The default start time for quiet time in ISO 8601 format. Required if `end` is set
+
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `application_id` - The Application ID of the Pinpoint App.
+
+## Import
+
+CodeBuild Project can be imported using the `application-id`, e.g.
```

Comment:
Copy-paste typo (`CodeBuild Project`) 😎

Reply:
Fixed in https://github.com/terraform-providers/terraform-provider-aws/pull/5956/commits/96756bf5c0cbb84b433765c3f0b76ac32261de06